.github/actions/issue-validator/package.json

```diff
@@ -2,13 +2,14 @@
   "private": true,
   "main": "dist/index.js",
   "scripts": {
-    "build": "ncc -o . -m build src/index.js --license licenses.txt"
+    "build": "ncc -m -o . build src/index.js --license licenses.txt"
   },
   "devDependencies": {
     "@vercel/ncc": "0.34.0"
   },
   "dependencies": {
     "@actions/core": "1.9.0",
-    "@actions/github": "5.0.3"
+    "@actions/github": "5.0.3",
+    "undici": "5.8.0"
```

Comment:
I think you can run this action with Node.js 18 to get the same behavior without a new dependency

Reply:
I tried setting Node 18 before, but I think it didn't work for some reason. I'll try again. 👍

Reply:
So the GitHub Action runner for JavaScript only supports Node 12 and 16:  https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing :shrug:

Given Node 16 support was announced in May https://github.blog/changelog/2022-05-20-actions-can-now-run-in-a-node-js-16-runtime/, Node 18 might take a long time, so I decided to not use an action just run it as a script. I tested locally with [act](https://github.com/nektos/act), and it seems to work. :crossed_fingers: